### PR TITLE
feat(Table): add copied tooltip text for table col element

### PR DESCRIPTION
**Pull request by GaroGabrielyan**

### CODE REVIEW

**Review by kirakosyangor — APPROVED** [review]

File: src/lib/organisms/Table/index.js [hunk 1]
```diff
@@ -663,6 +663,7 @@ Table.propTypes = {
             copyable: PropTypes.bool,
             exportDisabled: PropTypes.bool,
             copyTooltipText: PropTypes.string,
+            copiedTooltipText: PropTypes.string,
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/index.js:666]: For `copyTooltipText` and `copiedTooltipText` doc comments are absent, let's add documentation comments for these props.

File: src/lib/organisms/Table/Row/col.js [hunk 2]
```diff
@@ -97,7 +104,7 @@ function Col({
             >
                 <SkeletonLoader height={20} isBusy={guidRef.current && promiseValue === guidRef.current}>
                     {copyable && value && (
-                        <Tooltip title={copyTooltipText}>
+                        <Tooltip title={isCopied ? copiedTooltipText : copyTooltipText}>
                             <Icon
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/Row/col.js:108]: I suggest improving UX using the checkmark icon when the user clicks the copy icon.
It would be best if you had changed the icon type conditionally like the `copiedTooltipText` value. As a checkmark icon, I suggest using this one `bc-icon-checkbox-checked`.

https://github.com/softconstruct/gene-ui-components/assets/150047343/453d7ba7-a3ee-467e-8815-645c61ca4344

File: src/lib/organisms/Table/Row/col.js [hunk 3]
```diff
@@ -97,7 +104,7 @@ function Col({
...
                                 tabIndex="1"
                                 className="cursor-pointer copy-icon"
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/Row/col.js:110]: Prevent `handleCopy` execution when `isCopied` value `false` yet.

File: src/lib/organisms/Table/Row/col.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import { guid, stopEvent, copyToClipboard } from 'utils';
 import Icon from '../../../atoms/Icon';
 import SkeletonLoader from '../../../atoms/SkeletonLoader';
 import Tooltip from '../../../molecules/Tooltip';
+import callAfterDelay from '../../../../utils/callAfterDelay';
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/Row/col.js:9]: Use alias imports.

**Review by AregSoft — APPROVED** [review]

File: src/lib/organisms/Table/index.js [hunk 5]
```diff
@@ -634,6 +634,10 @@ Table.propTypes = {
      *
      * exportDisabled: Hide that column from export
      *
+     * copyTooltipText: Text for the tooltip when you hover on the element that can be copied.
+     *
+     * copiedTooltipText: Text for the tooltip when you already pushed copy button on the element that can be copied.
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/index.js:639]: Please change the pushed to pressed.

File: src/lib/organisms/Table/Row/col.js [hunk 6]
```diff
@@ -3,10 +3,10 @@ import classnames from 'classnames';
...
 
 import { guid, stopEvent, copyToClipboard } from 'utils';
-import Icon from '../../../atoms/Icon';
-import SkeletonLoader from '../../../atoms/SkeletonLoader';
-import Tooltip from '../../../molecules/Tooltip';
-import callAfterDelay from '../../../../utils/callAfterDelay';
+import Icon from 'src/lib/atoms/Icon';
+import SkeletonLoader from 'src/lib/atoms/SkeletonLoader';
+import Tooltip from 'src/lib/molecules/Tooltip';
+import callAfterDelay from 'src/utils/callAfterDelay';
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/Row/col.js:9]: This path is the absolute path of the `callAfterDelay ` module to change it to alias import change it to 
`import { callAfterDelay } from 'utils';`

File: src/lib/organisms/Table/Row/col.js [hunk 7]
```diff
@@ -108,8 +108,8 @@ function Col({
                             <Icon
                                 tabIndex="1"
                                 className="cursor-pointer copy-icon"
-                                type="bc-icon-copy-mirror"
-                                onClick={handleCopy}
+                                type={isCopied ? 'bc-icon-checkbox-checked' : 'bc-icon-copy-mirror'}
+                                onClick={!isCopied && handleCopy}
```
Comment (hamikhambardzumyan) [inline src/lib/organisms/Table/Row/col.js:112]: Why did you change your mind about the `pointer-events: none;` solution?

**Review by hamikhambardzumyan — CHANGES_REQUESTED** [review]

**Review by AregSoft — DISMISSED** [review]

**Review by NairaMisakyan — APPROVED** [review]

File: src/lib/organisms/Table/Row/col.js (hunk 7 above)
Comment (hamikhambardzumyan) [reply src/lib/organisms/Table/Row/col.js:111]: ok

**Review by hamikhambardzumyan — APPROVED** [review]
